.gitignore

```diff
@@ -3,9 +3,10 @@ characters
 training/datasets
 extensions/silero_tts/outputs/*.wav
 extensions/elevenlabs_tts/outputs/*.wav
-extensions/sd_api_pictures/outputs/*
 extensions/tortoise_tts*/tortoise/**/*
```

Comment:
I added 
extensions/*/outputs/*.*
extensions/*/outputs/*

I wonder if that works correctly

Reply:
My reasoning for not doing that is that it'll cause weird conflicts with the "output_goes_here.txt" files

Reply:
ahh.. but do we really need those?

Reply:
No, but I didn't want to be the one to remove them haha